Tesla.css

```diff
@@ -0,0 +1,233 @@
+body {
+    max-width: 1920px;
+    margin: 0 auto;
+    font-family: Montserrat, sans-serif;
+    font-weight: 600;
+    font-style: normal;
+}
+
+.header {
+    position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)